src/components/web/WordBashWeb.tsx

```diff
@@ -14,7 +14,9 @@ export class WordBashWeb extends React.Component<Props> {
   public render() {
     const { wbState } = this.props;
 
-    if (wbState.screen === WordBashScreen.HOME) {
+    if (wbState.screen === WordBashScreen.GAME && wbState.gameState) {
+      return <GameScreenWeb gameState={wbState.gameState} onQuit={() => wbState.quitGame()} />;
+    } else {
```

Comment:
remove this else statement and just return the home screen - put an empty line above it

Reply:
Requested changes made!